orchestrator/migrations/versions/schema/2023-07-17_165303a20fb1_customer_id_to_varchar.sql

```diff
@@ -0,0 +1,125 @@
+-- Must drop the materialized view and recreate, as you cannot edit it directly/any of its dependencies
+DROP MATERIALIZED VIEW IF EXISTS subscriptions_search;
+
+-- Alter customer_id type
+ALTER TABLE subscriptions
+    ALTER COLUMN customer_id TYPE VARCHAR;
+ALTER TABLE subscription_customer_descriptions
+    ALTER COLUMN customer_id TYPE VARCHAR;
+
+-- Recreate the view
+CREATE MATERIALIZED VIEW IF NOT EXISTS subscriptions_search AS WITH rt_info AS (
+    SELECT s.subscription_id,
+        concat_ws(
+            ', ',
+            string_agg(
+                rt.resource_type || ':' || siv.value,
+                ', '
+                ORDER BY rt.resource_type
+            ),
+            string_agg(
+                distinct 'subscription_instance_id' || ':' || si.subscription_instance_id,
+                ', '
+            )
+        ) AS rt_info
+    FROM subscription_instance_values siv
+        JOIN resource_types rt ON siv.resource_type_id = rt.resource_type_id
+        JOIN subscription_instances si ON siv.subscription_instance_id = si.subscription_instance_id
+        JOIN subscriptions s ON si.subscription_id = s.subscription_id
+    GROUP BY s.subscription_id
+),
+sub_prod_info AS (
+    SELECT s.subscription_id,
+        array_to_string(
+            ARRAY ['subscription_id:' || s.subscription_id,
+                                          'status:' || s.status,
+                                          'insync:' || s.insync,
+                                          'subscription_description:' || s.description,
+                                          'note:' || coalesce(s.note, ''),
+                                          'customer_id:' || s.customer_id,
+                                          'product_id:' || s.product_id],
+            ', '
+        ) AS sub_info,
+        array_to_string(
+            ARRAY ['product_name:' || p.name,
+                                          'product_description:' || p.description,
+                                          'tag:' || p.tag,
+                                          'product_type:', p.product_type],
+            ', '
+        ) AS prod_info
+    FROM subscriptions s
+        JOIN products p ON s.product_id = p.product_id
+),
+fi_info AS (
+    SELECT s.subscription_id,
+        string_agg(
+            fi.name || ':' || fi.value,
+            ', '
+            ORDER BY fi.name
+        ) AS fi_info
```

Comment:
I don't think there is any change to the view?

Reply:
There is no change to the view, it just has to be recreated since there is a change to the `customer_id` column in two of the referenced tables.